mk/checkconf.mk

```diff
@@ -31,6 +31,22 @@ define check-conf-h
 	fi
 endef
 
+define build-conf-in
+	$(q)set -e;						\
+	cnf="$(strip $(foreach var,				\
+		$(call cfg-vars-by-prefix,CFG_),		\
+		$(call cfg-make-variable,$(var))))";		\
+	mkdir -p $(dir $@);					\
+	echo "# auto-generated TEE configuration file" >$@.tmp; \
+	echo "# TEE version ${CFG_TEE_VERSION}" >>$@.tmp;	\
```

Comment:
Ah, my mistake. You need to double the $ so that the variable is evaluated by the shell, not by make. Then it should work.
`echo "# TEE version: $${CFG_TEE_VERSION:-(undefined)}"`